confirm/confirm.go

```diff
@@ -0,0 +1,36 @@
+package confirm
+
+import (
+	"bufio"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+
+	"github.com/fatih/color"
+)
+
+// AskForConfirmation asks the user for confirmation. A user must type in "yes" or "no" and
+// then press enter. It has fuzzy matching, so "y", "Y", "yes", "YES", and "Yes" all count as
+// confirmations. If the input is not recognized, it will ask again. The function does not return
+// until it gets a valid response from the user.
+func AskForConfirmation(s string) bool {
+	reader := bufio.NewReader(os.Stdin)
+
+	for {
+		fmt.Printf("%s [y/n]: ", color.YellowString(s))
+
+		response, err := reader.ReadString('\n')
+		if err != nil {
+			log.Fatal(err)
+		}
+
+		response = strings.ToLower(strings.TrimSpace(response))
+
+		if response == "y" || response == "yes" {
+			return true
+		} else if response == "n" || response == "no" {
```

Comment:
This could be cleaned up in a case, see below


```go
switch strings.ToLower(strings.TrimSpace(response)) {
	case "y", "yes":
		return true
	case "n", "no":
		return false
	}
}
```

Reply:
Done